entities/weapons/ix_hands.lua

```diff
@@ -164,6 +176,18 @@ function SWEP:Think()
 				self:DropObject()
 			else
 				local physicsObject = self.holdEntity:GetPhysicsObject()
+				local currentPlayerAngles = self:GetOwner():EyeAngles()
+				local client = self:GetOwner()
+
```

Comment:
I would add in a way of dropping the prop with IN_USE here like so 

```
if (client:KeyDown(IN_USE)) then
	self:DropObject()
	return
end
```

Reply:
I didn't add this because sometimes you want to place an item it into your inventory while you have it in your hands, this change would make it inconvenient to do so.